serum-390/soen390-team16-godzilla
# Added custom GODZILLA logo to the Readme, the wiki, and to the web app

# Description

- Adds some custom logos to the README.md and the wiki page (with a new sidebar).
- Also added the same branding to the web app

## How to Test

- Go to Code tab > click on "main" > switch branches to this branch and check out the new readme
- Check out the web app: 
  ```bash
  cd docker
  ./build-and-run.sh
  ```
- After running the above, browse to <http://localhost:8080/> to see the web app changes
